st2actions/st2actions/runners/pythonrunner.py

```diff
@@ -165,6 +166,9 @@ def _get_output_values(self, exit_code, stdout, stderr, timed_out):
         else:
             error = None
 
+        if "InvalidStatusException" in stderr:
```

Comment:
Hmm, I thought only time we will get `InvalidStatusException` in stderr is : https://github.com/StackStorm/st2/pull/2835/files#diff-f1f34011cf21592061e7b15f398f4e3fR135. i.e when action_status in python_action_wrapper.py is anything but bool or None. That is why i went with this comparison and have user defined exception. 

I will definitely keep an eye on the issues related to this change. Thank you for reviewing this and being patient!! :)